cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/messaging/aws/sqs/SqsQueueResource.java

```diff
@@ -118,12 +117,8 @@ private void doSendRequest(final SendMessageRequest sendMessageRequest) {
                 return amazonSqsClient.receiveMessage(receiveMessageRequest).getMessages();
             } catch (final AmazonServiceException amazonServiceException) {
                 logger.error("Error receiving SQS messages on queue:[" + queueName + "]", amazonServiceException);
-                // Ignore service errors (5xx) and hope SQS recovers
-                if (amazonServiceException.getErrorType().equals(ErrorType.Service)) {
-                    Thread.sleep(SQS_SERVICE_ERROR_PAUSE_MILLIS);
-                } else {
-                    throw amazonServiceException;
-                }
+                // Ignore all amazon errors and hope SQS recovers
```

Comment:
In this case it was an UnknownHostException so it will not go into this clause. I would prefer:

try {
//as before
}
catch(Exception e) {
logger.error(...)
Thread.sleep(...);
}

What do you think?